index.html

```diff
@@ -1,633 +1,715 @@
 <!DOCTYPE html>
 <html ng-app="weechat">
-  <head>
```

Comment:
why did you reindent everything? makes it very hard to see what's changed

Reply:
At one point I was stuck with a div and I hit a magic key combination in VS Code to fix all indentation, I shouldn't have done that. However a lot did change legitimatly too because it ended up in a new parent div. I changed the indentation of the head and body back to what it was now, this make is slightly better.